prow/cmd/hmac/main_test.go

```diff
@@ -653,3 +654,166 @@ func TestBatchOnboardNewTokenForRepos(t *testing.T) {
 		})
 	}
 }
+
+func TestHandleInvitation(t *testing.T) {
+	tests := []struct {
+		name          string
+		urivs         []github.UserRepoInvitation
+		uoivs         []github.UserOrgInvitation
+		errs          []error
+		newHMACConfig config.ManagedWebhooks
+		wantUrivs     []github.UserRepoInvitation
+		wantUoivs     []github.UserOrgInvitation
+		wantErr       bool
```

Comment:
Using booleans to indicate an expected error instead of substring matching or type matching has the issue that it has a lot of potential to rot over time if it ends up erroring for a different reason, resulting in the functionality breaking but the test still passing